apps/wallet/src/ui/app/staking/stake/ValidatorFormDetail.tsx

```diff
@@ -155,7 +159,7 @@ export function ValidatorFormDetail({
                                 weight="semibold"
                                 color="gray-90"
                             >
-                                {apy > 0 ? `${apy}%` : '--'}
+                                {apy === null ? '--' : `${apy}%`}
```

Comment:
Seems like this logic is really consistent, can we make a format function that we can share?